GVFS/GVFS.Service/Handlers/RequestHandler.cs

```diff
@@ -0,0 +1,96 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System.Runtime.Serialization;
+
+namespace GVFS.Service.Handlers
+{
+    public class RequestHandler

```

Comment:
Can you add a small class comment with a brief description of the purpose / responsibilities of this class?